### PR TITLE
Rm unused variables

### PR DESCRIPTION
**Pull request by mkuehbach**

### CODE REVIEW

File: src/pynxtools_em/parsers/image_tiff_zeiss.py [hunk 1]
```diff
@@ -70,70 +66,54 @@ def get_metadata(self, payload: str):
...
-                        )
+                        self.flat_dict_meta[line] = ureg.Quantity(tmp[0], ureg.degC)
                     elif len(tmp) == 2 and tmp[1] == "X":
-                        self.tmp["flat_dict_meta"][line] = ureg.Quantity(tmp[0])
+                        self.flat_dict_meta[line] = ureg.Quantity(tmp[0])
                     elif len(tmp) == 3 and tmp[1] == "K" and tmp[2] == "X":
-                        self.tmp["flat_dict_meta"][line] = (
-                            ureg.Quantity(tmp[0]) * 1000.0
-                        )
+                        self.flat_dict_meta[line] = ureg.Quantity(tmp[0]) * 1000.0
```
Comment (lukaspie) [inline src/pynxtools_em/parsers/image_tiff_zeiss.py:98]:
```
if len(tmp) == 1:
    if tmp[0] in ["On", "Yes"]:
        self.flat_dict_meta[line] = True
    elif tmp[0] in ["Off", "No"]:
        self.flat_dict_meta[line] = False
elif len(tmp) == 2:
    if tmp[1] == "°C":
        self.flat_dict_meta[line] = ureg.Quantity(tmp[0], ureg.degC)
    elif tmp[1] == "X":
        self.flat_dict_meta[line] = ureg.Quantity(tmp[0])
elif len(tmp) == 3 and (tmp[1], tmp[2]) == ("K", "X"):
    self.flat_dict_meta[line] = ureg.Quantity(tmp[0]) * 1000.0
```
cannot add suggestions here for some reason.

File: src/pynxtools_em/parsers/nxs_imgs.py [hunk 2]
```diff
@@ -54,30 +54,28 @@ def identify_image_type(self):
...
-            pe = PointElectronicTiffParser(self.file_path, self.entry_id)
-            pe.parse_and_normalize()
-            pe.process_into_template(template)
-        elif image_parser_type == "set_of_zipped_png_protochips":
-            axon = ProtochipsPngSetParser(self.file_path, self.entry_id)
-            axon.parse_and_normalize()
-            axon.process_into_template(template)
-        # add here further specific content (sub-)parsers for formats from other
-        # tech partner or other custom parsing of images
+        elif parser_type == "tiff_point_electronic":
```
Comment (lukaspie) [inline src/pynxtools_em/parsers/nxs_imgs.py:72]: lookup table

File: src/pynxtools_em/reader.py [hunk 3]
```diff
@@ -105,25 +105,25 @@ def read(
 
         print("Parse and map pieces of information within files from tech partners...")
         if len(case.dat) == 1:  # no sidecar file
-            images = NxEmImagesParser(entry_id, case.dat[0], verbose=False)
+            images = NxEmImagesParser(case.dat[0], entry_id, verbose=False)
```
Comment (lukaspie) [inline src/pynxtools_em/reader.py:108]: Again, don't repeat these things.
You can do 
```
parsers = [NxEmImagesParser, RsciioVeloxParser, ...]
for parser in parsers:
    parser= parsrer(case.dat[0], entry_id, verbose=False)
    parser.parse(template)
```

File: src/pynxtools_em/parsers/image_tiff_hitachi.py [hunk 4]
```diff
@@ -103,23 +101,24 @@ def check_if_tiff_hitachi(self):
...
                 if len(tmp) == 2 and all(token != "" for token in tmp):
                     try:
-                        self.tmp["flat_dict_meta"][tmp[0]] = ureg.Quantity(tmp[1])
+                        self.flat_dict_meta[tmp[0]] = ureg.Quantity(tmp[1])
                     except UndefinedUnitError:
-                        self.tmp["flat_dict_meta"][tmp[0]] = string_to_number(tmp[1])
+                        self.flat_dict_meta[tmp[0]] = string_to_number(tmp[1])
                     except TokenError:
-                        self.tmp["flat_dict_meta"][tmp[0]] = string_to_number(tmp[1])
+                        self.flat_dict_meta[tmp[0]] = string_to_number(tmp[1])
```
[the remark this answers is not in the document]
Comment (mkuehbach) [reply src/pynxtools_em/parsers/image_tiff_hitachi.py:113]: implemented

File: src/pynxtools_em/parsers/image_tiff_zeiss.py (hunk 1 above)
Comment (mkuehbach) [reply src/pynxtools_em/parsers/image_tiff_zeiss.py:98]: All good

File: src/pynxtools_em/parsers/nxs_imgs.py (hunk 2 above)
Comment (mkuehbach) [reply src/pynxtools_em/parsers/nxs_imgs.py:72]: Outdated, individual parsers have been moved to reader.py

**Pull request merged**